app/src/main/java/com/github/gouravkhunger/quotesapp/ui/adapters/SavedQuotesAdapter.kt

```diff
@@ -82,7 +94,9 @@ class SavedQuotesAdapter : RecyclerView.Adapter<SavedQuotesAdapter.QuoteViewHold
             rvAuthorTv.visibility = View.VISIBLE
 
             rvQuoteLoading.visibility = View.GONE
-
+            shareQuote.setOnClickListener {
+                clickListener.onShareClicked(holder.itemView)
+            }
```

Comment:
Similar thing here, please hide the share button before sending the view to share util class.